modules/rosa-sts-creating-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+[id="rosa-sts-creating-cluster_{context}"]
+= Creating your cluster with STS
+
+You can create an {product-title} cluster with STS using the `rosa` CLI.
```

Comment:
s/ `rosa` CLI / ROSA CLI (`rosa`)

Andrea recently requested that the CLIs be documented like this.